src/dotnet/commands/dotnet-help/LocalizableStrings.resx

```diff
@@ -268,6 +268,6 @@
     <value>Path to additional deps.json file.</value>
   </data>
   <data name="InstallDefinition" xml:space="preserve">
-    <value>Add item to development environment.</value>
+    <value>Installs an item into the development environment.</value>
```

Comment:
@KathleenDollard do you think this is a good change? I just copy pasted what you give me :)

Reply:
My thinking behind the change: match the verbs as to not confuse with the `add command`.  Also, the tense of the verb matches what most of the command descriptions use.

Can easily remove the commit if we don't think it is a valuable change.

Reply:
I think Peter is right on this...

To help future thinking on similar issues, my logic was initially that we don't want descriptions that repeat the name of the feature, because they add nothing. 

However, he is correct, that "add" had a different meaning in context, and is not really adding much beyond install. 

or put another way, his pragmatism trumps my theory.